src/main/java/com/faforever/client/ui/progress/ProgressCircleIndicator.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2014, Andrea Vacondio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.faforever.client.ui.progress;
+
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.ReadOnlyBooleanProperty;
+import javafx.beans.property.ReadOnlyBooleanWrapper;
+import javafx.beans.property.ReadOnlyIntegerProperty;
+import javafx.beans.property.ReadOnlyIntegerWrapper;
+import javafx.css.CssMetaData;
+import javafx.css.Styleable;
+import javafx.css.StyleableDoubleProperty;
+import javafx.css.StyleableProperty;
+import javafx.css.converter.SizeConverter;
+import javafx.scene.control.Control;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+
+/**
+ * Base class for the progress indicator controls represented by circualr shapes
+ *
+ * @author Andrea Vacondio
+ */
+abstract class ProgressCircleIndicator extends Control {
```

Comment:
Interesting will have to look at it but is a circular indicator that much better than the linear one we already have?

Also have to make sure it doesn't overload the client XD

Reply:
It is only updated every second. So no overload possible

Reply:
![Screenshot_20220915_192658](https://user-images.githubusercontent.com/19996713/192138195-bbe52584-afd2-457e-9bfc-f8c58affee92.png)

Reply:
Yeah that does look pretty nice